src/manager/hydraPaper.ts

```diff
@@ -1,23 +1,33 @@
+import Gio from 'gi://Gio';
+
 import * as Utils from '../utils.js';
+import {Settings} from './../settings.js';
 
 import {ExternalWallpaperManager} from './externalWallpaperManager.js';
 
+// https://gjs.guide/guides/gjs/asynchronous-programming.html#promisify-helper
+Gio._promisify(Gio.Subprocess.prototype, 'communicate_utf8_async', 'communicate_utf8_finish');
+
 /**
  * Wrapper for HydraPaper using it as a manager.
  */
 class HydraPaper extends ExternalWallpaperManager {
     protected readonly _possibleCommands = ['hydrapaper', 'org.gabmus.hydrapaper'];
 
+    /**
+     * We have to know if HydraPaper is in a version >= 3.3.2
+     * With that version the behavior changed to automatic light/dark mode detection.
+     */
+    private static _versionIsOld?: boolean;
```

Comment:
Honestly, I don't see a compelling point for keeping the `_` since the move to TS. You can leave the naming as it is in this PR, but in the future, don't add the underscore to new variables/functions (except there is some good reason to do so).

Reply:
Good point, that could be the case ... Maybe there is some easy way to add this to private variables during compilation in the future so we don't need to take care of this manually. Anyways, let's keep it as is for now. :)